designer/server/src/views/account/sign-in.njk

```diff
@@ -0,0 +1,29 @@
+{% extends 'layouts/page.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block content %}
+  {% if userFailedAuthorisation %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          html: "We couldn't sign you in. Please contact the system administrator for help, <a href=\"mailto:steven.thomas@defra.gov.uk\">steven.thomas@defra.gov.uk</a>."
```

Comment:
Couple of tiny text + link class tweaks

```suggestion
          html: "We couldn’t sign you in. Please contact the system administrator for help, <a href=\"mailto:steven.thomas@defra.gov.uk\" class="govuk-link">steven.thomas@defra.gov.uk</a>."
```

<br>

But ideally, can we use the [`appErrorSummary()` component](https://github.com/DEFRA/forms-designer/commit/24c87625eaba1f64fa68657e91286cd88ac02ff0) wrapper I made?

```njk
{% from "error-summary/macro.njk" import appErrorSummary %}
```

I've extended it for form creation errors that come from the API, but something like this:

* https://github.com/DEFRA/forms-designer/pull/146

Otherwise we mainly need to notify screen readers of the error

## Page title
First we'll need to suffix the page title with `Error: `

```njk
{% if userFailedAuthorisation | length %}
  {% set pageTitle = "Error: " + pageTitle %}
{% endif %}
```

## HTML block `safe` filter
But good to use HTML blocks via `safe` too, here's an example via `appErrorSummary()`

```njk
  {% set errorHtml %}
    We couldn’t sign you in. Please contact the system administrator for help,
    <a href="mailto:steven.thomas@defra.gov.uk" class="govuk-link">steven.thomas@defra.gov.uk</a>
  {% endset %}

  {% if userFailedAuthorisation %}
    {{ appErrorSummary({
      errorList: [{
        html: errorHtml | safe
      }]
    }) }}
  {% endif %}
```

Reply:
I've merged https://github.com/DEFRA/forms-designer/pull/146 now if you'd like to rebase

Reply:
Done